src/core/server/http_resources/http_resources_service.ts

```diff
@@ -76,8 +79,12 @@ export class HttpResourcesService implements CoreService<InternalHttpResourcesSe
     const cspHeader = deps.http.csp.header;
     return {
       async renderCoreApp(options: HttpResourcesRenderOptions = {}) {
+        const apmConfig = getApmConfig(request.url.pathname);
         const body = await deps.rendering.render(request, context.core.uiSettings.client, {
           includeUserSettings: true,
+          vars: {
+            apmConfig,
+          },
```

Comment:
We were not including the `apmConfig` variable when rendering an app via `renderCoreApp` and/or `renderAnonymousCoreApp`.

As the `/app/{id}` route is now also using `renderCoreApp`, I added it. (also, I assumed we just forgot to do so)